src/pages/AccountAddress.vue

```diff
@@ -161,74 +231,278 @@ export default {
+                }
+                this.hash = this.subtab;
+                this.menus[menu] = true;
+                this.indicators[menu] = true;
+                this.$refs.panels.goTo(this.subtab);
+            }
+        },
         disableConfirmation(){
             this.confirmationDialog = false;
         },
+        getIcon,
+        formatWei,
+        toChecksumAddress,
     },
 };
 </script>
 
 <template>
-<div class="pageContainer q-pt-xl">
+<div v-if="accountAddress && !accountLoading" :key="accountAddress" class="pageContainer q-pt-xl">
     <div>
         <div class="row tableWrapper justify-between q-mb-lg">
             <div class="homeInfo">
-                <div class="text-primary text-h4 q-pr-xs">
-                    {{ title }}
-                </div>
-                <q-icon
-                    v-if="isContract && isVerified !== null"
-                    class="cursor"
-                    :name="isVerified ? 'verified' : 'warning'"
-                    :class="isVerified ? 'text-positive' : 'text-negative'"
-                    size="1.25rem"
-                    @click="confirmationDialog = true"
-                />
-                <ConfirmationDialog
-                    class="text-secondary"
-                    :flag="confirmationDialog"
-                    :address="address"
-                    :status="isVerified"
-                    @dialog="disableConfirmation"
-                />
-                <CopyButton
-                    class="text-secondary"
-                    :text="address"
-                    :accompanyingText="address"
-                    description="address"
-                />
-                <template v-if="contract">
-                    <div class="text-white">
-                        {{ $t('pages.created_at_trx' )}} &nbsp;
-                        <TransactionField :transaction-hash="contract.getCreationTrx()"/>
-                    </div>
-                    <div class="text-white">{{ $t('pages.by_address') }}
-                        &nbsp;
-                        <AddressField :address="contract.getCreator()"/>
-                    </div>
-                </template>
-                <small v-else>
-                    <div class="text-white">
-                        {{ $t('pages.number_used_once') }}: &nbsp;
-                        <span class="q-pl-xs">{{ nonce }}</span>
-                    </div>
-                </small>
-            </div>
-            <div class="dataCardsContainer">
-                <div v-if="!!telosAccount" class="dataCardItem">
-                    <div class="dataCardTile">
-                        {{ $t('pages.native_account') }}
+                <div class="flex">
+                    <q-img
+                        v-if="this.contract?.supportedInterfaces.includes('erc20')"
+                        class="coin-icon"
+                        :alt="this.contract.getName() + ' ERC20 token'"
```

Comment:
in general, it can cause problems if you use `this` keyword in templates. it should be like `:alt="contract.getName() + ' ERC20 token'"` as the `this` is automatically inferred. so that should be changed in this whole file's template, as well as in any other templates changed